docs/documentation/ui-rest-api.md

```diff
@@ -172,6 +172,27 @@ Sample response:
 }
 ```
 
+### /api/v1/history/summary (GET)
+
+Returns a list of topology ID submitted by the current user.
+
+Response fields:
+
+|Field  |Value | Description|
+|---	|---	|---
+|topo-history| List| List of Topologies' IDs|
+
+Sample response:
+
+```json
+{
+    "topo-history":[
+        {"host":"wc6-1-1446571009"},
```

Comment:
Yes, nice catch. This is legacy code, actually, we only need to return a list of topo-id, the key "host" does not make sense (it is just a hard-coded string, not real host name). I will fix it.